kyo-prelude/shared/src/main/scala/kyo/kernel/Frame.scala

```diff
@@ -21,7 +21,7 @@ object Frame:
     ) derives CanEqual:
 
         def show: String =
-            Ansi.highlight(s"// $declaringClass $methodName", snippetLong, s"// $position", position.lineNumber)
+            Ansi.highlight(s"// $position $declaringClass $methodName", snippetLong, s"", position.lineNumber)
```

Comment:
I've moved the file position to the header and left the footer empty to make the rendering easier to follow.